applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.h

```diff
@@ -111,7 +111,17 @@ int le_audio_octets_per_frame_get(const struct bt_audio_codec_cfg *codec, uint32
  */
 int le_audio_frame_blocks_per_sdu_get(const struct bt_audio_codec_cfg *codec,
 				      uint32_t *frame_blks_per_sdu);
-/*
+
+/**
+ * @brief	Decode the audio frame duration and number of octets per frame in the codec
```

Comment:
"Decode" can confuse the reader. Suggest replacing with "get".

Reply:
I re-wrote it a bit so the brief says "get bitrate", and the details contain the "decode other stuff"

Reply:
Great.I still believe the use of the word "decode" can confuse